.github/workflows/test.yml

```diff
@@ -40,5 +41,8 @@ jobs:
         with:
           node-version: "${{ matrix.node-version }}"
           cache: npm
-      - run: npm ci
+      - run: npm clean-install
       - run: npm run lint
+      # https://github.com/lirantal/lockfile-lint#readme
+      - name: Scan lockfile for security issues
+        run: npx lockfile-lint --path package-lock.json
```

Comment:
for the other repos with more recent changes, we've moved this into the scripts in `package.json` that end up getting called from `npm t`

this changeset is a good reference for the group of changes that are related to what i'm highlighting here. i would also be ok with these changes being a follow-up to this PR, but the changes you are making in this PR set us up for those changes to make sense whichever way you decide is best

Reply:
after a little more thought, i think it may make sense to hold off on changing this for a follow-up instead of adding into this one. still open to either option, but i think splitting from this isolates things a little better

Reply:
Follow-Up PR it is ✌️

Reply:
for reference: https://github.com/semantic-release/npm/blob/bf03ab46877800195fb45a6d2987e18efbab3e08/package.json#L109-L112